Xamarin.PropertyEditing.Tests/PanelViewModelTests.cs

```diff
@@ -58,7 +58,6 @@ public async Task InteruptedEditorRetrievalResolvesCorrectlyItemAdded ()
 			await Task.Yield ();
 
 			Assert.That (vm.Properties, Is.Empty);
-			this.context.ThrowPendingExceptions ();
```

Comment:
I'm assuming this is the correct change as the ThrowPendingExceptions should be happening in the TearDown and now here.

Reply:
Exactly!